core/include/kernel/interrupt.h

```diff
@@ -32,6 +32,12 @@
 
 #define ITRF_TRIGGER_LEVEL	(1 << 0)
 
+#define cpumask_of(cpu)		(1 << cpu)
```

Comment:
One problem here where the CPU number is from, `get_core_pos()` is not guaranteed to give the number compatible with the GIC view. When reading the GIC spec it looks like the CPU numbering is implementation (platform) specific.